routes/web.php

```diff
@@ -80,167 +78,197 @@
+            Route::post('/people/{contact}/relationships/{otherContact}', 'Contacts\\RelationshipsController@update')->name('update');
+            Route::delete('/people/{contact}/relationships/{otherContact}', 'Contacts\\RelationshipsController@destroy')->name('delete');
+        });
 
         // Pets
-        Route::get('/people/{contact}/pets', 'Contacts\\PetsController@get');
-        Route::post('/people/{contact}/pet', 'Contacts\\PetsController@store');
-        Route::put('/people/{contact}/pet/{pet}', 'Contacts\\PetsController@update');
-        Route::delete('/people/{contact}/pet/{pet}', 'Contacts\\PetsController@trash');
-        Route::get('/petcategories', 'Contacts\\PetsController@getPetCategories');
+        Route::name('pets.')->group(function () {
+            Route::get('/people/{contact}/pets', 'Contacts\\PetsController@get')->name('index');
+            Route::post('/people/{contact}/pet', 'Contacts\\PetsController@store')->name('store');
+            Route::put('/people/{contact}/pet/{pet}', 'Contacts\\PetsController@update')->name('update');
+            Route::delete('/people/{contact}/pet/{pet}', 'Contacts\\PetsController@trash')->name('delete');
+            Route::get('/petcategories', 'Contacts\\PetsController@getPetCategories');
+        });
 
         // Reminders
-        Route::get('/people/{contact}/reminders/add', 'Contacts\\RemindersController@create')->name('.reminders.add');
-        Route::post('/people/{contact}/reminders/store', 'Contacts\\RemindersController@store')->name('.reminders.store');
-        Route::get('/people/{contact}/reminders/{reminder}/edit', 'Contacts\\RemindersController@edit')->name('.reminders.edit');
-        Route::put('/people/{contact}/reminders/{reminder}', 'Contacts\\RemindersController@update')->name('.reminders.update');
-        // Special route to delete reminders. In one migration in summer '17, I
-        // accidentely f**ked up the reminders table by messing up the contact ids
-        // and now the only way to delete those reminders is to bypass the ReminderRequest
-        // by creating a new route.
-        Route::delete('/people/{contact}/reminders/{rmd}', 'Contacts\\RemindersController@destroy')->name('.reminders.delete');
+        Route::name('reminders.')->group(function () {
+            Route::get('/people/{contact}/reminders/add', 'Contacts\\RemindersController@create')->name('add');
+            Route::post('/people/{contact}/reminders/store', 'Contacts\\RemindersController@store')->name('store');
+            Route::get('/people/{contact}/reminders/{reminder}/edit', 'Contacts\\RemindersController@edit')->name('edit');
+            Route::put('/people/{contact}/reminders/{reminder}', 'Contacts\\RemindersController@update')->name('update');
+            Route::delete('/people/{contact}/reminders/{reminder}', 'Contacts\\RemindersController@destroy')->name('delete');
```

Comment:
@djaiss Is the problem you described always here (Special route to delete reminders) ? I replaced the route, and all is OK.

Reply:
@asbiin really? Well if it works now, awesome.